docs/user/accumulo/configuration.rst

```diff
@@ -1,26 +1,39 @@
-Configuration
--------------
+Accumulo Configuration
+======================
 
-GeoMesa Accumulo uses a site XML file to maintain system property configurations. This file can be found
-at ``conf/geomesa-site.xml`` of the Accumulo distribution. The default settings for GeoMesa are
-stored in ``conf/geomesa-site.xml.template``. Do not modify this file directly as it is never read;
-instead copy the desired configurations into ``geomesa-site.xml``.
+This section details Accumulo specific configuration properties. For general properties,
+see :ref:`geomesa_site_xml`.
 
-By default, system properties set through command line parameters will take precedence over this
-configuration file. If you wish a configuration item to always take precedence, even over command
-line parameters, change the ``<final>`` tag to true.
+Batch Writer Properties
+-----------------------
 
-By default configuration properties with empty values will not be applied, you can change this
-by marking a property as final.
+The following properties control the configuration of Accumulo ``BatchWriters``. They map directly to the
```

Comment:
[picky] writing:

    ``BatchWriter``\ s

will let you do a plural without including the "s" in the class name.